src/utils/store-odp-data.js

```diff
@@ -0,0 +1,150 @@
+const redis = require('redis')
+const request = require('request-promise-native')
+const { promisify } = require('util')
+
+async function redisClientConnect() {
+  return redis.createClient({
+    port: process.env.REDIS_PORT,
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD
+  })
+}
+
+async function getDatasets() {
+  let rawJsonData = await request(process.env.OPEN_DATA_URL)
+  const rawData = JSON.parse(rawJsonData)
+  return rawData.dataset.map(function(entry) {
+    return { label: entry.title, identifier: entry.identifier }
+  })
+}
+
+function sortOrder(a, b) {
+  return a.label < b.label ? -1 : a.label > b.label ? 1 : 0
```

Comment:
Use an if statement here. Nested ternaries are a headache to read and an even bigger headache to debug.